Add powerview supported shade information (#22944)
home-assistant/home-assistant.github.io: source/_integrations/hunterdouglas_powerview.markdown

{% include integrations/config_flow.md %}

## Supported Shades

Known devices and their supported functionality is listed below.

> :warning: Devices listed as **Under Development** may perform basic or slightly odd functionality until **Full** support is implemented.\
> We ask you wait patiently for Full support before raising support tickets for other models.\
> \
> If however you're shade is not listed below please raise a support request for them to be added.

> | Type | Common Name                        | Capabilities                    | Support           |

Reviewer: Generally we don't want to get support requests in the issue queue.  We only want people to open an issue if there is an actual bug of flaw.  It would be best to avoid any language that implies its a place where they can go to get support as we want to send people to the community forums for support.

Author: Happy to reword that to direct them to the feature requests of the forum ?

Reviewer: If its a feature request we send users to https://community.home-assistant.io/c/feature-requests. If its support https://discord.gg/c5DvZ4e or the forums

But in this case they should find the urls elsewhere since they may change in the future and we don't want to maintain them here as well. 

Ideally we just avoid saying that support is available or something is supported as it implies that the issue queue is a support queue which we want to avoid.  Usually I try to say things are `tested` or `confirmed working`

Author: how do i tell what release the tdbu suport will be in - ie the table will need to be updated for the correct next branch

Author: rewording commited now

| 54   | Vertical Slats, Left Stack         | 180° Tilt when closed, Vertical | Under Development |
| 55   | Vertical Slats, Right Stack        | 180° Tilt when closed, Vertical | Under Development |
| 56   | Vertical Slats, Split Stack        | 180° Tilt when closed, Vertical | Under Development |
| 62   | Venetian, Tilt Anywhere            | 180° Tilt Anywhere, Vertical    | Under Development |
| 65   | Vignette Duolite                   | Dual Shade Blackout             | Under Development |
| 69   | Curtain, Left Stack                | 180° Tilt when closed, Vertical | Under Development |
| 70   | Curtain, Right Stack               | 180° Tilt when closed, Vertical | Under Development |
| 71   | Curtain, Split Stack               | 180° Tilt when closed, Vertical | Under Development |
| 79   | Duolite Lift                       | Dual Shade Blackout             | Under Development |

## Capabilities Information

### Bottom Up

These shades offer only the simple up/down movement of your conventional shades.

### Top Down

> :information_source: Full functionality not currently implemented

> These shades offer a unique movement which is inverse to your conventional shade where the shade is fixed to the floor and lowers from the roof.

Reviewer: ```suggestion
These shades offer a unique movement that is inverse to your conventional shade, where the shade is fixed to the floor and lowered from the roof.
```

### Top Down, Bottom Up (TDBU)

> :information_source: Full functionality not currently implemented
TDBU shades consist of two rails controlled by two motors designated by Top and Bottom with fabric in between.

> The Top and Bottom can move independently of each other to cover different parts of the window but cannot pass the other.

Reviewer: ```suggestion
The Top and Bottom can move independently to cover different parts of the window but cannot pass the other.
```

Two different entities that will be created for each blind: Top and Bottom.

#### Top entity

> - 'Up/Open' will move the Top rail to the bottom of the window and set the Bottom rail to it's closed position.

Reviewer: ```suggestion
- 'Up/Open' will move the top rail to the bottom of the window and set the bottom rail to its closed position.
```

Reviewer: Or make them all `Top` and `Bottom`, but its confusing to have both

Author: I think they were all right to begin with - couldnt find any to fix.

When referring to the shades i used capatilised letters and when referring to the window it was lower case as i was indicating an item vs a position

ie - out of the 3 below the first felt the most natural - please let me know if you want all Top rail references changed - Perhaps it would read better with Rail capatilised too ?

Move the Top rail to the top of the window
Move the Top rail to the Top of the window
Move the top rail to the top of the window

Reviewer: It's not a sticking point for me, only a suggestion. I'm happy with any of them.

Author: ok - if you want to take another run over it all your corrections are done

> - 'Down/Close' will move the Top rail to the top of the window and set the Bottom rail to it's closed position.

Reviewer: ```suggestion
- 'Down/Close' will move the top rail to the top of the window and set the bottom rail to its closed position.
```

Reviewer: Or make them all `Top` and `Bottom`, but its confusing to have both

> - 'Position' is the position in which the Top rail can move, so from the top of the window (0) to the bottom of the window (100).

Reviewer: ```suggestion
- 'Position' is the position in which the top rail can move from the top of the window (0) to the bottom (100).
```

Reviewer: Or make them all `Top` and `Bottom`, but its confusing to have both

> - Note as the position of a Top rail cannot pass that of a Bottom rail if you set an 'impossible position' the position will fall back to the closest possible value to that requested.

Reviewer: ```suggestion
- The position of a top rail cannot pass that of a bottom rail. If you set an impossible position, the position will fall back to the closest possible value to that requested.
```

Reviewer: Or make them all `Top` and `Bottom`, but its confusing to have both

#### Bottom entity

> - 'Up/Open' will move the Bottom rail to the top of the window and set the Top rail to it's closed position.

Reviewer: ```suggestion
- 'Up/Open' will move the bottom rail to the top of the window and set the top rail to its closed position.
```

Reviewer: Or make them all `Top` and `Bottom`, but its confusing to have both

> - 'Down/Close' will move the Bottom rail to the bottom of the window and set the Top rail to it's closed position.

Reviewer: ```suggestion
- 'Down/Close' will move the bottom rail to the bottom of the window and set the Top rail to its closed position.
```

Reviewer: Or make them all `Top` and `Bottom`, but its confusing to have both

> - 'Position' is the position in which the Bottom rail can move, so from the bottom of the window (0) to the top of the window (100).

Reviewer: ```suggestion
- 'Position' is the position in which the bottom rail can move from the bottom (0) to the top of the window (100).
```

Reviewer: Or make them all `Top` and `Bottom`, but its confusing to have both

> - Note as the position of a Bottom rail cannot pass that of a Top rail if you set an 'impossible position' the position will fall back to the closest possible value to that requested.

Reviewer: ```suggestion
- The position of a bottom rail cannot pass that of a top rail. If you set an impossible position, the position will fall back to the closest possible value to that requested.
```

Reviewer: Or make them all `Top` and `Bottom`, but its confusing to have both

### Tilt when closed

> Shades with Tilt with Tilt when closed functionality only allow a shade to tilt when it is closed and will automatically close the vanes when the shade is moved from it's closed position.

Reviewer: Not sure how to clean this one up `Shades with Tilt with Tilt when closed` seems like a typo?

Author: fixed

### Tilt Anywhere

> :information_source: Full functionality not currently implemented

> These shades can tilt in any position and do not require the shade to be open or closed to adjust tilt position.

Reviewer: ```suggestion
These shades can tilt in any position and do not require the shade to be open or closed to adjust the tilt position.
```

### Dual Shade Blackout

> :information_source: Full functionality not currently implemented

> These shades consist of two pieces of fabric attached to a single rail. The front shade is sheer with the rear being blackout and neither panel can move independently to the other.

Reviewer: ```suggestion
These shades consist of two pieces of fabric attached to a single rail. The front shade is sheer, with the rear being blackout, and neither panel can move independently from the other.
```

{% include integrations/config_flow.md %}

## Shades

> ### Known working devices and their functionality is listed below.

Reviewer: ```suggestion
### Known working devices
```

<div class="note">
If your shade is not listed please raise a feature request on the community forum.
</div>

| Name (Type)                            | Capabilities                    |
| :------------------------------------- | :------------------------------ |
| Roman (4)                              | Bottom Up                       |
| Bottom Up (5)                          | Bottom Up                       |
| Duette (6)                             | Bottom Up                       |
| Silhouette (23)                        | 90° Tilt when closed            |
| M25T Roller Blind (42)                 | Bottom Up                       |
| Twist (44)                             | 180° Tilt when closed           |
| AC Roller  (49)                        | Bottom Up                       |

> ### Devices under with limited functionality

Reviewer: ```suggestion
### Devices with limited functionality
```

{% include integrations/config_flow.md %}

> ## Supported Shades

Reviewer: ```suggestion
## Shades
```

{% include integrations/config_flow.md %}

## Shades

> Known devices and their supported functionality is listed below.

Reviewer: ```suggestion
### Known working devices and their functionality is listed below.
```

<div class="note">
If your shade is not listed please raise a feature request on the community forum.
</div>

| Type and Name                            | Capabilities                    |
| :--------------------------------------- | :------------------------------ |

> | 4   - Roman                              | Bottom Up                       |
> | 5   - Bottom Up                          | Bottom Up                       |
> | 6   - Duette                             | Bottom Up                       |
> |  23 - Silhouette                         | 90° Tilt when closed            |
> |  42 - M25T Roller Blind                  | Bottom Up                       |
> |  44 - Twist                              | 180° Tilt when closed           |
> |  49 - AC Roller                          | Bottom Up                       |

Reviewer: ```suggestion
| Roman (4)                             | Bottom Up                       |
| Bottom Up (5)                          | Bottom Up                       |
| Duette (6)                            | Bottom Up                       |
| Silhouette (23)                        | 90° Tilt when closed            |
| M25T Roller Blind (42)                 | Bottom Up                       |
| Twist (44)                              | 180° Tilt when closed           |
| AC Roller  (49)                         | Bottom Up                       |
```

I think most people won't know the product type number so maybe that should come after the name (same below)

> 

Reviewer: ```suggestion

### Devices under with limited functionality

```
